spec/queries/dashboard/timeline_document_query_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Dashboard::TimelineDocumentQuery do
+  describe '.call' do
+    let(:admin) { create(:user, :admin) }
+    let(:expected_document) { create(:document, :cv, creator: admin, prospect: create(:prospect, target: true)) }
+    let(:not_expected_document) { expected_document }
+
+    it 'Only documents added by administrators in last 5 days for target prospects with only CVs and team presentation' do
+      expect(Dashboard::TimelineDocumentQuery.call).to match_array([expected_document])
```

Comment:
it does not test for `team_presentation`... also consider not using `let` (stevo way :P)